build.gradle

```diff
@@ -1,103 +1,188 @@
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+        exclude group: 'com.fasterxml.jackson.datatype', module: 'jackson-datatype-jdk8'
+        exclude group: 'javax.validation', module: 'validation-api'
+    }
+
+    compile "org.embulk:embulk-util-json:0.1.1"
+    compile "org.embulk:embulk-util-text:0.1.1"
+    compile "org.embulk:embulk-util-timestamp:0.2.1"
+
+    // Explicit dependencies for embulk-util-* that matches with Embulk
+    compile 'com.fasterxml.jackson.core:jackson-core:2.6.7'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.6.7'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.7'
+    compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.7'
+    compile 'javax.validation:validation-api:1.1.0.Final'
+
+    compile('org.embulk:embulk-base-restclient:0.10.1') {
+        exclude group: 'org.msgpack', module: 'msgpack-core'
+    }
+    compile('org.embulk:embulk-util-retryhelper-jetty92:0.8.2') {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
+    }
+
+    compile 'com.google.guava:guava:18.0'
+    compile  "com.google.code.findbugs:annotations:3.0.1"
+    compile 'org.apache.commons:commons-lang3:3.4'
+
     testCompile "junit:junit:4.+"
-    testCompile "org.embulk:embulk-core:0.8.+:tests"
-    testCompile "org.embulk:embulk-test:0.8.+"
+    testCompile "org.embulk:embulk-core:$embulkVersion:tests"
+    testCompile "org.embulk:embulk-junit4:$embulkVersion"
+    testCompile "org.embulk:embulk-deps:$embulkVersion"
     testCompile "org.mockito:mockito-core:2.+"
 }
 
-task classpath(type: Copy, dependsOn: ["jar"]) {
-    doFirst { file("classpath").deleteDir() }
-    from (configurations.runtime - configurations.provided + files(jar.archivePath))
-    into "classpath"
+embulkPlugin {
+    mainClass = "org.embulk.input.marketo.MarketoInputPlugin"
+    category = "input"
+    type = "marketo"
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = project.group
+            artifactId = project.name
+
+            from components.java  // Must be "components.java". The dependency modification works only for it.
+            // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+            // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                name = project.name
+                description = project.description
+                url = "https://www.embulk.org/"
+
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "MIT License"
```

Comment:
i think this is Apache License

Reply:
The project was initialized with an MIT license. See LICENSE.txt